src/Microsoft.SqlTools.ServiceLayer/DisasterRecovery/DisasterRecoveryService.cs

```diff
@@ -110,9 +111,21 @@ public void InitializeService(ServiceHost serviceHost)
                 SqlConnection sqlConn = GetSqlConnection(connInfo);
                 if (sqlConn != null)
                 {
+                    // initialize backup
                     DisasterRecoveryService.Instance.InitializeBackup(helper.DataContainer, sqlConn);
                     DisasterRecoveryService.Instance.SetBackupInput(backupParams.BackupInfo);
-                    DisasterRecoveryService.Instance.PerformBackup();
+
+                    // create task metadata
+                    TaskMetadata metadata = new TaskMetadata();
+                    metadata.ServerName = connInfo.ConnectionDetails.ServerName;
+                    metadata.DatabaseName = connInfo.ConnectionDetails.DatabaseName;                    
+                    metadata.Name = "Backup Database";
+                    metadata.Description = "Backup Database";
+                    metadata.IsCancelable = true;
+
+                    // create backup task and perform
+                    SqlTask sqlTask = TaskService.Instance.TaskManager.CreateTask(metadata, Instance.BackupTask);
+                    sqlTask.Run();                    
```

Comment:
that is my fault to not have a sync method to call, it's not a good practice to call an async method but not wait for it. is it possible for you to add a sync method in Task service and call that? or create a sync method here and then call task.Run?